os/sdk-modifying-coreos.md

```diff
@@ -168,7 +168,19 @@ After `build_image` completes, it prints commands for converting the raw bin int
 
 Once you build an image you can launch it with KVM (instructions will print out after `image_to_vm.sh` runs).
 
-If you encounter errors with KVM, verify that it is supported by your CPU and enabled in the BIOS.
+If you encounter errors with KVM, verify that virtualization is supported by your CPU by doing `egrep '(vmx|svm)' /proc/cpuinfo`. The `/dev/kvm` directory will be in your host OS when virtualization is enabled in the BIOS.
+
+The `./coreos_production_qemu.sh` file can be found in the `~/trunk/src/build/images/amd64-usr/latest` directory of the booted filesystem.
+
+#### Several Ways to Boot:
+
+After `image_to_vm.sh` completes, do `./coreos_production_qemu.sh -curses` to launch a graphical interface to login to the Container Linux vm.
+
+You could instead use the `-nographic` option, `./coreos_production_qemu.sh -nographic` , which gives you the ability to switch from the vm to the Qemu monitor console by pressing <kbd>CTRL</kbd>+<kbd>a</kbd> and then <kbd>c</kbd>. To exit out of login, press <kbd>CTRL</kbd>+<kbd>a</kbd> and then <kbd>x</kbd>.
+
+You could also login via SSH keys by doing `./coreos_production_qemu.sh -a ~/.ssh/authorized_keys -p 2223 -- -curses`. The `-p` option is used to provide a different port number. Refer to the [Booting with Qemu](https://coreos.com/os/docs/latest/booting-with-qemu.html#SSH-keys) guide for more information on this usage.
```

Comment:
s/login/log in/
s/doing/running/

This language implies that `-a` and `-p` are necessary to use SSH keys, which is not the case.